scripts/github-conventional-comments.js

```diff
@@ -42,6 +42,17 @@
       'Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.',
   };
 
+  const EMOJI = {
+    [LABEL.praise]: '\u{1F44F}',
+    [LABEL.note]: '\u{1F5D2}',
+    [LABEL.nitpick]: '\u{1F913}',
+    [LABEL.thought]: '\u{1F4AD}',
+    [LABEL.suggestion]: '\u{270F}',
+    [LABEL.question]: '\u{2753}',
+    [LABEL.chore]: '\u{2611}',
+    [LABEL.issue]: '\u{26A0}',
+  };
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice work -- great use of ChatGPT!‏